java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -21,10 +21,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.nio.charset.StandardCharsets;
+import java.io.*;
```

Comment:
I generally prefer not to have .* imports unless there are a huge number of them.

Reply:
Agree, intellij barfed on me with imports.